src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,44 @@
+package duke.commands;
+
+import duke.exceptions.RepeatedTaskException;
+import duke.tasks.Task;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * The AddCommand task is a Command that adds a task to a list of tasks.
+ */
+public class AddCommand extends Command {
+    Task task;
+
+    /**
+     * Constructor - Initialises the task to be added.
+     *
+     * @param task input task
```

Comment:
As per the coding standard, how about
```suggestion
     * @param task input Task to be added.
```